### ISSUE TITLE
No way to generate metadata when not using the admin ui

### ISSUE THREAD

**Issue by jstebenne** [matt-j-m/grav-plugin-aura#17]
I avoid using the admin UI to keep my installation simple and content edition quick. However, there is no way for me to generate post metadata from the CLI when I edit markdown files.

It would be nice to have a CLI tool to bulk generate those metadata. An admin UI button to do it could also be possible when this tool is developped.

**Comment by sherpadawan** [comment]
I totally agree onAdminSave procedure should be accessible by means of a button in the plugin blueprint form which could allow to bulk generate all metadata for the whole existing pages.
Then a CLI tool could be also a great thing too.

**Comment by sherpadawan** [comment]
Would be very usefull to distinguish dev/prod servername as well !
If a cli is accessible then a crontab could be setup
